skfda/representation/basis.py

```diff
@@ -374,6 +380,10 @@ def _ndegenerated(self, penalty_degree):
         """
         return penalty_degree
 
+    def _derivative(self, coefs, order=1):
+        return (self.copy(), coefs if order == 0
```

Comment:
I am not sure if we allow the coefficients to be changed, but if we do, this should be a copy. Maybe even is better that a copy is always passed to this function in that case, to prevent possible mistakes.